src/utils/webrtc/MediaDevicesManager.js

```diff
@@ -403,8 +403,9 @@ MediaDevicesManager.prototype = {
 		this._stopIncompatibleTracks(constraints)
 
 		return navigator.mediaDevices.getUserMedia(constraints).then(stream => {
-			this._registerStream(stream)
-
+			const videoEffect = new VideoEffects()
```

Comment:
I wonder if we should move this to LocalMedia instead.
The LocalMedia class has state variables like "video enabled", "audio enabled" so we could add a new attribute "videoEffectsEnabled" and "setVideoEffectsEnabled()" where we can handle whatever we need to do to reset the streams.

If we do so, there are at least three locations there where `mediaDevicesManager.getUserMedia().then(function (stream) {}` is called, so we could write a new private function that wraps the stream if necessary.

Reply:
That sounds good. Somehow I missed that spot. (my first tests even were in localmedia)